src/routes/user.routes.js

```diff
@@ -14,7 +14,7 @@ import {
 const users = express.Router();
 
 users.post('/', adminProtect, celebrate({ [Segments.BODY]: addUserSchema }), tracedAsyncHandler(create));
-users.get('/', adminProtect, tracedAsyncHandler(getAll));
+users.get('/', externalPartyProtect, adminProtect, tracedAsyncHandler(getAll));
```

Comment:
Somewhere below there's a user update route as well I guess that needs restriction as well